stream_alert_cli/helpers.py

```diff
@@ -635,28 +631,45 @@ def user_input(requested_info, mask, input_restrictions):
 def load_test_file(path):
     """Helper to json load the contents of a file with some error handling
 
+    Test files can be either formatted as:
+
+    {
+        "records": [
+            {"data": {}, "description": "", ...}
+        ]
+    }
+
+    or
+
+    [
+        {"data": {}, "description": "", ...}
+    ]
+
     Args:
         path (str): Relative path to file on disk
 
     Returns:
-        dict: Loaded JSON from test event file
+        list: Loaded JSON from test event file
     """
     message_template = 'Improperly formatted file ({}): {}'
     with open(path, 'r') as test_event_file:
         try:
             contents = json.load(test_event_file)
         except (ValueError, TypeError) as err:
             message = message_template.format(path, err.message)
-            return False, message
-
-        # Make sure the test event file is formatted in the way we expect
-        if not (isinstance(contents, dict) and 'records' in contents):
-            message = message_template.format(path, 'File must be a dict (JSON '
-                                              'object) with top level key \'records\'')
-
-            return False, message
+            return [], message
+        else:
+            # Check for legacy format, return a list
```

Comment:
Let's log an error (or warning at least) here mentioning that the events should be updated to the new format, and add a TODO that flags this for removal sometime in the future. we should probably discuss creating a specific `TECHDEBT` (or similar) todo tag that we can search for in the future.